pwa/src/templates/templateParts/header/HeaderTemplate.module.css

```diff
@@ -40,3 +40,12 @@
 .languageSelectDisabled {
   opacity: 50%;
 }
+
+.visuallyHidden {
+  position: absolute !important;
+  height: 1px;
+  width: 1px;
+  overflow: hidden;
+  clip: rect(1px, 1px, 1px, 1px);
+  color: red;
+}
```

Comment:
Might be good to create a component for this purpose. Right now, when we want multiple hidden elements for navigational purposes, we would need to re-write this code.

Out of scope for this pull request, I think. But maybe nice for the backlog?

Reply:
Do we need the `color` rule?